addons/account/models/account_move.py

```diff
@@ -215,7 +215,8 @@ def _get_default_invoice_incoterm(self):
         copy=False, readonly=True,
         domain=[('exclude_from_invoice_tab', '=', False)],
         states={'draft': [('readonly', False)]})
-    invoice_partner_bank_id = fields.Many2one('res.partner.bank', string='Bank Account',
+    invoice_partner_bank_id = fields.Many2one('res.partner.bank', string='Bank Account', store=True, readonly=False,
+        compute="_compute_invoice_partner_bank_id",
```

Comment:
Lets keep related arguments together, and lets reduce the amount of lines modified
```suggestion
    invoice_partner_bank_id = fields.Many2one('res.partner.bank', string='Bank Account',
        compute="_compute_invoice_partner_bank_id", store=True, readonly=False,